toffy/qc_metrics_plots.py

```diff
@@ -0,0 +1,75 @@
+from ark.utils.load_utils import load_imgs_from_tree, load_imgs_from_dir
+import matplotlib.pyplot as plt
+import seaborn as sns
+import numpy as np
+from toffy import qc_comp
+from ark.utils import io_utils
+import pandas as pd
+
+
+def call_violin_swarm_plot(plotting_df, fig_label, figsize=(20, 3), fig_dir=None):
+    """Makes violin plot with swarm dots. Used with make_batch_effect_plot()
+
+    Args: pandas df with "sample", "channel", "tma", "99.9th_percentile"
```

Comment:
Please make sure to follow the syntax we use for docstrings, for example here
https://github.com/angelolab/toffy/blob/main/toffy/normalize.py